csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html

```diff
@@ -0,0 +1,186 @@
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="{% static 'tech_fair_css.css' %}"/>
+	<title>SFU Tech Fair 2022</title>
+
+
+  <script>
```

Comment:
this problem is back 
![Screenshot from 2022-09-05 14-55-49](https://user-images.githubusercontent.com/14242088/188515528-3bfff192-1734-433c-bcfd-1ade7eb588e7.png)
![Screenshot from 2022-09-05 14-55-58](https://user-images.githubusercontent.com/14242088/188515529-0c918304-c668-4974-a70a-9fc6ca3e9306.png)